AmplifyPlugins/Auth/AWSCognitoAuthPluginTests/AuthUserBehaviorTests/UserBehaviorChangePasswordTests.swift

```diff
@@ -149,7 +149,7 @@ class UserBehaviorChangePasswordTests: BaseUserBehaviorTest {
     /// - When:
     ///    - I invoke changePassword with old password and new password
     /// - Then:
-    ///    - I should get a .requestLimitExceeded error
+    ///    - I should get a .limitExceeded error
     ///
     func testChangePasswordWithLimitExceededException() {
```

Comment:
Add test for `requestLimitExceeded` exception also.

Reply:
There is a `testChangePasswordWithTooManyRequestsException` in the tests. Similarly, in other files changed there is a test for `requestLimitExceeded` available.